src/test/java/com/github/terma/jenkins/githubprcoveragestatus/SonarMasterCoverageRepositoryTest.java

```diff
@@ -73,6 +73,7 @@ public void should_get_zero_coverage_for_unknown_metric() throws IOException {
     private void givenCoverageRepository() {
         buildLogOutputStream = new ByteArrayOutputStream();
         sonarMasterCoverageRepository = new SonarMasterCoverageRepository("http://localhost:" + wireMockRule.port(),
+                "myToken",
                 new PrintStream(buildLogOutputStream, true));
```

Comment:
Please add at least two test cases here, one to demonstrate that in case of ```sonarToken = null``` we don't include any additional to ```request``` and test case for non empty